### DISCUSSION TITLE
Figma files for Keycloak/Patternfly

### DISCUSSION THREAD

**Discussion by paultempleman** [keycloak/keycloak#28340] [Ideas]
With Keycloak moving to Patternfly I was wondering if there is any possibility of the Keycloak UI team publishing Figma files for Keycloak like the Patternfly team have done at **https://www.figma.com/community/file/1300826511900225842/patternfly-5**

**Comment by jonkoops** [comment]
We don't have our whole UI in Figma designs, so there would be little purpose. We simply re-use existing PatternFly components to build our UI to our needs, with little to no customization.
